Jenkinsfile_CNP

```diff
@@ -41,13 +41,21 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
   ]
 }
 
-
 withPipeline(type, product, component) {
-  onPR {
-    log.info(env.CHANGE_TITLE)
-    env.CHANGE_TITLE = '[PREVIEW] ' + env.CHANGE_TITLE
-    log.info(env.CHANGE_TITLE)
-  }
-  enableSlackNotifications('#sscs-tech')
+  enableDockerBuild()
+  installCharts()
   loadVaultSecrets(secrets)
+  enableSlackNotifications('#sscs-tech')
+  enableAksStagingDeployment()
+  nonServiceApp()
+
+  before('akschartsinstall') {
+    echo "Setting up data for functional tests"
+    gradle(functionalPreDeploy)
```

Comment:
you should be using gradleBuilder:
https://github.com/hmcts/bulk-scan-processor/blob/master/Jenkinsfile_CNP#L64
`GradleBuilder builder = new GradleBuilder(this, product)`
`builder.gradle('functionalPreDeploy')`